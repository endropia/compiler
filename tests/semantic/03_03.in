type point = record x: integer; y: integer; end;
var
	a: point;
begin end.