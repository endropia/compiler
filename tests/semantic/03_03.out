program : Unnamed program
   alias
      record
      x
         type: integer
      y
         type: integer
      point
   var: 
      a
      type: point
   stmts:
      empty
scope     name                          class               
------------------------------------------------------------
0         integer                       primitive type      
0         double                        primitive type      
0         boolean                       primitive type      
0         char                          primitive type      
0         string                        primitive type      
1         point                         alias               
1         a                             variable            
